src/components/general/SideSheet/Modal/index.tsx

```diff
@@ -10,21 +10,25 @@ import {
 } from '@equinor/fusion-components';
 import { useComponentDisplayClassNames } from '@equinor/fusion';
 
+export type SideSheetSize = 'xlarge' | 'large' | 'medium' | 'small';
+
 type ModalSideSheetProps = {
     children: ReactNode;
     header?: string;
     show?: boolean;
     onClose?: () => void;
     headerIcons?: ReactNode[];
+    size?: SideSheetSize;
```

Comment:
Maybe we should default to xLarge so existing modal side sheets don't unexpectedly change size

Reply:
default to xLarge